<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QMainWindow" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>IDDRRA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="14" column="7">
     <widget class="QPushButton" name="PB_onlyAnalysis">
      <property name="text">
       <string>Go to Analysis  </string>
      </property>
     </widget>
    </item>
    <item row="21" column="7">
     <widget class="QPushButton" name="PB_nxt">
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </item>
    <item row="21" column="3">
     <widget class="QPushButton" name="PB_runDNA_Design">
      <property name="text">
       <string>Run DNA design</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select output filename:</string>
      </property>
     </widget>
    </item>
    <item row="21" column="4">
     <widget class="QPushButton" name="PB_cancel">
      <property name="text">
       <string>cancel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QGraphicsView" name="GV_image"/>
    </item>
    <item row="20" column="1">
     <widget class="QLineEdit" name="LE_pyOutput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="20" column="2">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>.pdb</string>
      </property>
     </widget>
    </item>
    <item row="21" column="1" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="PB_openScript">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Open DNA Design script</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="PB_editDatFile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Open .dat file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIDDRA">
    <property name="title">
     <string>IDDRRA</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuIDDRA"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
